test/e2e/petset.go

```diff
@@ -212,6 +212,64 @@ var _ = framework.KubeDescribe("StatefulSet [Slow] [Feature:PetSet]", func() {
 			pst.verifyPodAtIndex(updateIndex, ps, verify)
 		})
 
+		It("scaling down before scale up is finished should wait until current pod will be running and ready before it will be removed", func() {
+			By("creating petset " + psName + " in namespace " + ns)
```

Comment:
`"Creating stateful set " + psName + " in namespace " + ns + ", and pausing scale operations after each pod"`